src/resource.rc

```diff
@@ -51,7 +51,8 @@ BEGIN
 	POPUP "&Song"
 	BEGIN
 		MENUITEM "&Play\tCtrl+P", ID_PLAY
-		MENUITEM "&Stop\tEsc", ID_STOP
+		MENUITEM "&Stop\tEsc", ID_STOP, GRAYED
+		MENUITEM "Capture &Audio", ID_CAPTURE_AUDIO
```

Comment:
```suggestion
		MENUITEM "Capture &Audio...", ID_CAPTURE_AUDIO
```

Since it opens a separate dialog to open a file. (Ditto for the instance in `stop_capturing_audio`)